src/components/frame/v5/pages/BalancePage/partials/BalanceTable/hooks.tsx

```diff
@@ -92,6 +92,11 @@ export const useBalanceTableColumns = (
               />
               <CurrencyConversion
                 tokenBalance={currentTokenBalance}
+                tokenDecimals={
+                  row.original.token
+                    ? getTokenDecimalsWithFallback(row.original.token.decimals)
+                    : 0
+                }
```

Comment:
This should just be 
```suggestion
                tokenDecimals={
                  getTokenDecimalsWithFallback(row.original.token.decimals)
                }
```

Since `getTokenDecimalsWithFallback` with fall back to `18` even if the first argument is `null`/`undefined`.

And in any case, `0` is bad value for decimals